src/components/ChangeEmailInApp.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from "react";
```

Comment:
Whole new ChangeEmailInApp file to do change email functionality, still needs formatting, it is in progress, will be easier to complete after merge